apps/evm/src/hooks/useOperationModal/Modal/WithdrawForm/index.tsx

```diff
@@ -237,6 +256,29 @@ const WithdrawForm: React.FC<WithdrawFormProps> = ({ asset, pool, onCloseModal }
     vToken: asset.vToken,
   });
 
+  const nativeTokenGatewayContractAddress = useGetNativeTokenGatewayContractAddress({
+    comptrollerContractAddress: pool.comptrollerAddress,
+  });
+
+  const { mutateAsync: updatePoolDelegateStatus, isLoading: isUseUpdatePoolDelegateStatusLoading } =
+    useUpdatePoolDelegateStatus({
+      delegateAddress: nativeTokenGatewayContractAddress || '',
+      poolComptrollerAddress: pool.comptrollerAddress,
+    });
+
+  const { data: isDelegateApproved, isLoading: isDelegateApprovedLoading } =
+    useGetPoolDelegateApprovalStatus(
+      {
+        poolComptrollerAddress: pool.comptrollerAddress,
+        delegateAddress: nativeTokenGatewayContractAddress!,
```

Comment:
I'd match the way you pass `delegateAddress` parameter to the `useUpdatePoolDelegateStatus` hook:
```suggestion
        delegateAddress: nativeTokenGatewayContractAddress || '',
```